apps/SageAccountWeb/src/components/RegisterAccount1.tsx

```diff
@@ -56,6 +56,20 @@ export const RegisterAccount1 = () => {
   const [membershipInvitationEmail, setMembershipInvitationEmail] =
     useState<string>()
 
+  const { search } = useLocation()
+  const queryParams = useMemo(() => new URLSearchParams(search), [search])
+  const emailFromParams = queryParams.get('email')
+
+  // If we have an email param, initialize the email address with the param
+  useEffect(() => {
+    if (emailFromParams) {
+      setEmail(emailFromParams)
+      setPage(Pages.EMAIL_REGISTRATION)
+    }
+    // Initialize the email address field with the email query parameter, but allow the user to change it or register using OAuth
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [])
```

Comment:
I wonder if this should unconditionally set the email (to blank if emailFromParams is undefined), as it might fix the browser trying to prefill the field.